docsets/Vulkan/docset.json

```diff
@@ -1,10 +1,20 @@
 {
     "name": "Vulkan",
-    "version": "1.0.38",
+    "version": "1.0.39",
     "archive": "Vulkan.tgz",
     "author": {
         "name": "Lei Zhang",
         "link": "https://github.com/antiagainst"
     },
-    "aliases": ["vk", "GLNext", "Graphics API"]
+    "aliases": ["vk", "GLNext", "Graphics API"],
+    "specific_versions": [
+        {
+            "version": "1.0.39",
+            "archive": "versions/1.0.39/Vulkan.tgz"
+        },
+        {
+            "version": "1.0.38",
+            "archive": "versions/1.0.38/Vulkan.tgz"
```

Comment:
What probably beneficial is a readme/wiki page describing the procedure of updating docsets. :)